src/test/overlay/reduce_relay_test.cpp

```diff
@@ -1058,7 +1058,7 @@ class reduce_relay_test : public beast::unit_test::suite
                 bool mustHandle = false;
                 if (event.state_ == State::On)
                 {
-                    assert(event.key_);
+                    BEAST_EXPECT(event.key_);
```

Comment:
Let's use our old friend `if (BEAST_EXPECT(event.key_))` here, so that it doesn't get dereferenced on the off chance that it's not set.